app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/StringComparator.java

```diff
@@ -0,0 +1,44 @@
+package openfoodfacts.github.scrachx.openfood.utils;
+
+import java.text.Collator;
+import java.text.Normalizer;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Locale;
+
+/**
+ * Created by Raymond Chenon on 05/05/20.
+ * Compare the strings ignoring the accents on letters ([a-z][A-Z])
+ */
+public class StringComparator implements Comparator<String> {
+
+    public static <T> void sort(T[] arrays) {
+        Arrays.sort(arrays, Collator.getInstance(Locale.US));
```

Comment:
Change `Locale.US` to the current Android language: `Locale.getDefault()`.

Reply:
Let's keep `Locale.US`, because it strips all accents.

`Locale.getDefault()` will be unpredictable because each language can have its own rules for how strings and letters are sorted.

http://tutorials.jenkov.com/java-internationalization/collator.html

Reply:
We should follow how each language would sort the items, don't you think?
@raychenon
cc. @teolemon

Reply:
That's basically why I suggested collator. My understanding is that people took the pain to do the sorting right in every single language.

Reply:
Ok, now the `Collator` has to be passed to set explicitly the Locale. Otherwise, it could be hard to debug due to `Locale.getDefault()`

Didn't notice any difference between Locale.US, Locale.FR (no Locale.ES).